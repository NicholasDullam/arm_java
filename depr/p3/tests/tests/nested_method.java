class Base {
    public static int method2(int a) {
        System.out.println(a);
        return 7;
    }
    
    public static int method1(int b) {
        System.out.println(b);
        int x = method2(7);
        return b - x;
    }

	public static void main(String[] args){
        System.out.println(args[0]);
        int a = method1(10);
        System.out.println(a);
	}
}
