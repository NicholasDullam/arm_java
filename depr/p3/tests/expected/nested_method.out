745
10
7
3
